packages/components/pie-modal/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
```

Comment:
This can be deleted and the entries added to your changeset MD file.

The CHANGELOG will get generated automatically once this PR + Verison Packages PR is merged 🎉